README.md

```diff
@@ -53,6 +53,8 @@ To get started with using Wagtail, run the following in a virtual environment:
 ![Installing Wagtail](.github/install-animation.gif)
 
 ```sh
+python -m venv venv
```

Comment:
Issue: Above, we already mention that these commands should be run in a virtualenv - therefore it's misleading to create another.

Perhaps we should instead just reference some other docs on virtualenvs?